cypress/integration/inventory/create-holdings-as-different-user.spec.js

```diff
@@ -36,6 +37,11 @@ describe('ui-inventory: Create a Holdings record as another user than the one th
   });
```

Comment:
Please pass the path parameter  into the login function instead of separate function cy.visit